modules/appeals_api/app/controllers/appeals_api/v0/appeals_controller.rb

```diff
@@ -7,6 +7,8 @@
 module AppealsApi
   module V0
     class AppealsController < ApplicationController
+      include AppealsApi::GatewayOriginCheck
```

Comment:
This was our only Appeals controller that didn't inherit from `AppealsApi::ApplicationController`, so the `before_action` didn't previously apply to it, but I believe this was a mistake and it was always intended to, so I have added it here.